cypress/support/elements/admin_settings_page_elements.js

```diff
@@ -0,0 +1,9 @@
+
+export const adminSettingsPageElements = {
```

Comment:
I was trying to enable default class settings for student. But it's not working as I thought it would.
So, deleted the tests for now. Will understand the feature better and update these tests with that settings.
But I am keeping the page objects as we can use it later.